### PR TITLE
News section on front page

### PR DESCRIPTION
**Pull request by gonuke**
Added news section to capture specific items - will require curation.

### CODE REVIEW

File: source/index.rst [hunk 1]
```diff
@@ -33,6 +29,16 @@ always welcome and encouraged to use or contribute to the |cyclus| project.
 
 The |Cyclus| project repository is located at http://github.com/cyclus/cyclus.
 
+News
+-----
+
+**August 13, 2015:** :doc:`Cyclus included in FY2016 workscopes for the NEUP program <news/fy16neup>`
+
+**July 21-24, 2015:** Join workshop with CLASS_ project held in Madison, WI
```
Comment (gidden) [inline source/index.rst]: s/join/joint

File: source/index.rst [hunk 2]
```diff
@@ -33,6 +29,16 @@ always welcome and encouraged to use or contribute to the |cyclus| project.
...
+
+**June 10, 2015:** :doc:`First Cyclus user tutorials are held at the 2015 ANS Annual Meeting in San Antonio, TX <user/tutorial/index>`
```
Comment (gidden) [inline source/index.rst]: do we also want to link to the dev tutorial?

**Comment by gidden** [comment]
looks good; a few comments inline

**Comment by gonuke** [comment]
ping - I'd like to get this up for possible NEUP traffic

**Comment by mbmcgarry** [comment]
looks good, thanks @gonuke !

**Pull request merged**